test/e2e/self_hosted_test.go

```diff
@@ -74,7 +75,8 @@ func TestCreateSelfHostedClusterWithBootMember(t *testing.T) {
 	defer e.Close()
 
 	<-e.Server.ReadyNotify()
-	fmt.Println("etcdserver is ready")
+
+	t.Log("etcdserver is ready")
```

Comment:
t.Log() will only show after the test finished.
This is not really helpful: 1. debugging; 2. mixing logs with others that already use more real-time logging like fmt, logrus.

Can we keep fmt? I don't mind to change to logrus.